### PR TITLE
CSS code improvements

### PR DESCRIPTION
**Pull request by juhilsomaiya**
Css code fixes.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,211 +1,210 @@
...
-  -ms-transition: all 0.7s ease-in-out;
-  transition: all 0.7s ease-in-out;
-}
-#stbar:hover
-{  
+  -moz-transition: all .7s ease-in-out;
+  -webkit-transition: all .7s ease-in-out;
+  -o-transition: all .7s ease-in-out;
+  -ms-transition: all .7s ease-in-out;
+  transition: all .7s ease-in-out
```
Comment (krman009) [inline style.css:20]: We can remove the prefixes as per [Can I use](https://caniuse.com/#feat=css-transitions).
Please remove the lines: 20 to 23

File: style.css [hunk 2]
```diff
@@ -1,211 +1,210 @@
...
+  background: -webkit-linear-gradient(top, rgba(255, 0, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(0, 255, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(255, 0, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(255, 255, 0, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1));
+  background: -ms-linear-gradient(top, rgba(255, 0, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(0, 255, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(255, 0, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(255, 255, 0, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1));
+  background: -o-linear-gradient(top, rgba(255, 0, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(0, 255, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(255, 0, 255, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1), rgba(255, 255, 0, .3), rgba(8, 180, 80, .1), rgba(180, 80, 80, .1));
```
Comment (krman009) [inline style.css]: Remove lines: 31, 32 as we do not need this anymore.

File: style.css [hunk 3]
```diff
@@ -1,211 +1,210 @@
...
+
+li {
+  display: inline-block
+}
+
+li a {
+  background: -moz-linear-gradient(bottom, rgba(0, 0, 0, .1), rgba(0, 0, 0, .3), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000);
+  background: -webkit-linear-gradient(bottom, rgba(0, 0, 0, .1), rgba(0, 0, 0, .3), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000);
+  background: -o-linear-gradient(bottom, rgba(0, 0, 0, .1), rgba(0, 0, 0, .3), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000);
+  background: -ms-linear-gradient(bottom, rgba(0, 0, 0, .1), rgba(0, 0, 0, .3), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000, rgba(0, 0, 0, .2), rgba(0, 0, 0, .1), #000);
```
Comment (krman009) [inline style.css]: Remove lines: 100 and 101

File: style.css [hunk 4]
```diff
@@ -1,211 +1,210 @@
...
+  font-size: 24px
+}
+
+li a:hover,
+li:hover {
+  -moz-transition: .5s;
+  transition: .5s;
+  -webkit-transition: .5s;
+  -o-transition: .5s;
+  -ms-transition: .5s
```
Comment (krman009) [inline style.css]: Remove Lines: 110, 112, 113 and 114.

File: style.css [hunk 5]
```diff
@@ -1,211 +1,210 @@
...
+
+li:hover {
+  transform: translateY(-5px)
+}
+
+li a:hover {
+  background: -moz-linear-gradient(bottom, rgba(110, 110, 110, .1), rgba(110, 110, 110, .3), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e);
+  background: -webkit-linear-gradient(bottom, rgba(110, 110, 110, .1), rgba(110, 110, 110, .3), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e);
+  background: -o-linear-gradient(bottom, rgba(110, 110, 110, .1), rgba(110, 110, 110, .3), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e);
+  background: -ms-linear-gradient(bottom, rgba(110, 110, 110, .1), rgba(110, 110, 110, .3), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e, rgba(110, 110, 110, .2), rgba(110, 110, 110, .1), #6e6e6e);
```
Comment (krman009) [inline style.css]: Remove lines: 124, 125

File: style.css [hunk 6]
```diff
@@ -1,211 +1,210 @@
...
-{
-  font-size: 17px;
-  color: orange;
-}
-.rm
-{
+  background: -moz-radial-gradient(top, #009 10%, transparent 80%);
+  background: -webkit-radial-gradient(top, #009 10%, transparent 80%);
+  background: -o-radial-gradient(top, #009 10%, transparent 80%);
+  background: -ms-radial-gradient(top, #009 10%, transparent 80%);
```
Comment (krman009) [inline style.css]: Remove last two lines.

File: style.css [hunk 7]
```diff
@@ -1,211 +1,210 @@
...
   color: #181818;
-  -moz-transition: 0.5s;
-  -o-transition: 0.5s;
-  -webkit-transition: 0.5s;
-  -ms-transition: 0.5s;
-  transition: 0.5s;
+  -moz-transition: .5s;
+  -o-transition: .5s;
+  -webkit-transition: .5s;
+  -ms-transition: .5s;
```
Comment (krman009) [inline style.css]: Remove prefixed lines.

File: style.css [hunk 8]
```diff
@@ -1,211 +1,210 @@
...
-.im
-{
-  padding: 7px 40px 5px;
-}
-.foot
-{
+  background: -moz-radial-gradient(top, #009 10%, transparent 80%);
+  background: -webkit-radial-gradient(top, #009 10%, transparent 80%);
+  background: -o-radial-gradient(top, #009 10%, transparent 80%);
+  background: -ms-radial-gradient(top, #009 10%, transparent 80%);
```
Comment (krman009) [inline style.css]: Remove last two lines too.

**Review by krman009 — CHANGES_REQUESTED** [review]
Please push with requested changes, I'll merge it ;) @juhilsomaiya

File: style.css (hunk 8 above)
Comment (juhilsomaiya) [reply style.css]: @krman009 can you please check ?

**Review by krman009 — APPROVED** [review]
Thanks buddy @juhilsomaiya :+1:

**Pull request merged**